tests/integrations_setup/install_aws_asset_inventory_integration.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+This script installs Asset Inventory AWS integration
+
+The following steps are performed:
+1. Create an agent policy.
+2. Create a Asset Inventory AWS integration.
+3. Render aws-asset-inventory.sh file to be used to install and enroll agent on an EC2
+instance
+"""
+import sys
+from pathlib import Path
+
+import configuration_fleet as cnfg
+from fleet_api.agent_policy_api import create_agent_policy
+from fleet_api.common_api import (
+    get_artifact_server,
+    get_enrollment_token,
+    get_fleet_server_host,
+    get_package_version,
+)
+from fleet_api.package_policy_api import create_integration
+from fleet_api.utils import render_template
+from loguru import logger
+from munch import Munch
+from package_policy import (
+    VERSION_MAP,
+    generate_random_name,
+    load_data,
+    version_compatible,
+)
+from state_file_manager import HostType, PolicyState, state_manager
+
+EXPECTED_AGENTS = 1
+PKG_DEFAULT_VERSION = VERSION_MAP.get("asset_inventory_aws", "")
+INTEGRATION_NAME = "Asset Inventory AWS"
+INTEGRATION_INPUT = {
+    "name": generate_random_name("pkg-asset-inventory-aws"),
+    "input_name": "asset_inventory_aws",
+    "vars": {
+        "access_key_id": cnfg.aws_config.access_key_id,
+        "secret_access_key": cnfg.aws_config.secret_access_key,
+        "aws.credentials.type": "direct_access_keys",
+    },
+}
+AGENT_INPUT = {
+    "name": generate_random_name("asset-inventory-aws"),
+}
+script_template = Path(__file__).parent / "data/cspm-linux.j2"
+
+
+if __name__ == "__main__":
+    # pylint: disable=duplicate-code
+    package_version = get_package_version(cfg=cnfg.elk_config)
```

Comment:
`get_package_version` by default returns the version of the `cloud_security_posture` integration. I believe that asset inventory uses a different package name: `cloud_asset_inventory`.

Reply:
Nice nice, good catch.